### PR TITLE
Header / Footer - Connie

### PR DESCRIPTION
**Pull request by connietee22**
Here's the first basic draft of the header + footer -- without our fancy fonts!

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -6,4 +6,112 @@ html { -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing:
...
+    color: #fff;
+    text-shadow: 3px 2px 1px #b20062;
+    padding: 20px;
+}
+
+header {
+    height: 80vh;
+    display: flex;
+    align-items: flex-end;
+    background-image: linear-gradient(to bottom,rgba(255, 225, 249, .1), rgba(10, 10, 10, .2)), url('./assets/thomas-park-unsplash.png');
```
Comment (kdlry) [inline src/App.css:63]: Nice background image, love the purple mug!

File: src/App.css [hunk 2]
```diff
@@ -29,6 +31,51 @@ ol {
...
+    background-repeat: no-repeat;
+    background-position: bottom;
+    padding-bottom: 100px;
+}
+
+.caffeine {
+    display: block;
+}
+
+.animate__bounceIn {
```
Comment (afacey) [inline src/App.css:75]: Cool animation!

**Comment by kdlry** [comment]
Looks good, ready to merge!

**Comment by raphaelroques** [comment]
conflicts resolved. ready to push

**Pull request merged**